db.go

```diff
@@ -287,5 +287,5 @@ func (dbImpl *DBImpl) rounRobin(n int) int {
 	if n <= 1 {
 		return 0
 	}
-	return int(1 + (atomic.AddUint64(&dbImpl.replicasCount, 1) % uint64(n)))
```

Comment:
I see... This is missed. Because of this the round robin only start from index 1.